### PR TITLE
Added nuget.config to root of the repo to help dotnet restore run wit…

### PR DESCRIPTION
**Pull request by OrrinEdenfield**
See discussion on this PR: https://github.com/Azure-Samples/azure-iot-samples-csharp/pull/238

### CODE REVIEW

**Review by drwill-ms — APPROVED** [review]

File: nuget.config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
```
Comment (drwill-ms) [inline nuget.config:4]: Pipeline indicated a need for a line before this:

> nuget.config - Missing <clear/> statement.


```xml
    <clear />
```

**Review by drwill-ms — CHANGES_REQUESTED** [review]
See my code comment.

**Comment by OrrinEdenfield** [comment]
Line added to file.

**Review by drwill-ms — APPROVED** [review]

**Pull request merged**